### DISCUSSION TITLE
Problem:  Documentation needs refresh

### DISCUSSION THREAD

**Discussion by radiorob** [openbroadcaster/observer#31] [Ideas]
Could use a hand documenting

Server setup

Provisioning Players

**Comment by coryshort** [comment]
I will be doing a server reinstall over the next week i see if i can document it as i go.

**Comment by gosntvrdi** [comment]
> I will be doing a server reinstall over the next week i see if i can document it as i go.

did you manage to document it?

**Comment by coryshort** [comment]
not yet something came up so it got shelved for a bit

**Comment by maxlinux2000** [comment]
Hello
I can write some help documentation when I get this program to work. At the moment I am planning to modify the installation script so that it works without problems in ubunu 20.04 and debian10 buster .... then I have to take notes on how to program a show, because it is not clear and I would like it to be more intuitive.
But it is what it is and we have to make this media server work.

I was thinking of writing a "Quick Start" document without going too far into settings and options that I don't quite understand.

But for this I will have to ask you many questions :-) Are you ready?

**Comment by radiorob** [comment]
I've been doing some updating and rearranging of our support site, to assemble the material so it is easier to get to.  I'll make an item for a Quick Start Guide.  You can find these at support.openbroadcaster.com

**Comment by Sturmey66** [comment]
The getting started documentation and the various videos on the youtube channels don't address the issue of what do you actually need to get started. I've read almost everything I could find and I'm still not 100% sure of what I need to install and get working to have this project function.

Also the naming of the components seems counter intuitive. The "player" appears to be the stream originating server, and the "server" appears to be a control interface. 

One other thing, I can't seem to find reliable data on what the output stream options are. Can I output MP4-TS on a UDP stream? Can I ouput straight to HLS and have that input into my CDN? Also do you support resolutions higher than 720p? I'm looking for software that can handle 4k video, but I can't find information on if this will do it.
